spark/src/test/scala/org/apache/comet/CometExpressionSuite.scala

```diff
@@ -2122,6 +2122,22 @@ class CometExpressionSuite extends CometTestBase with AdaptiveSparkPlanHelper {
     }
   }
 
+  test("pad") {
+    val table = "pad"
+    val gen = new DataGenerator(new Random(42))
+    withTable(table) {
+      // generate some data
+      // newline characters are intentionally omitted for now
+      val dataChars = "\t abc123"
+      sql(s"create table $table(id int, name varchar(20)) using parquet")
+      gen.generateStrings(100, dataChars, 6).zipWithIndex.foreach { x =>
+        sql(s"insert into $table values(${x._2}, '${x._1}')")
```

Comment:
Can we cover the same cases as https://github.com/apache/datafusion-comet/blob/main/spark/src/test/scala/org/apache/comet/CometExpressionSuite.scala#L2115
especially those unicode cases?